lncfg/db.go

```diff
@@ -266,6 +275,88 @@ func (db *DB) GetBackends(ctx context.Context, chanDBPath,
 			Remote:     true,
 			CloseFuncs: closeFuncs,
 		}, nil
+
+	case PostgresBackend:
+		postgresBackend, err := kvdb.Open(
+			kvdb.PostgresBackendName, ctx,
+			db.Postgres, NSChannelDB,
+		)
+		if err != nil {
+			return nil, fmt.Errorf("error opening postgres graph "+
+				"DB: %v", err)
+		}
+		closeFuncs[NSChannelDB] = postgresBackend.Close
+
+		postgresMacaroonBackend, err := kvdb.Open(
+			kvdb.PostgresBackendName, ctx,
```

Comment:
Ah ok I see where the namespaces are used (for the diff DBs), but imo we still should have a top-top level prefix to allow multiple `lnd` nodes to share the same database.

Reply:
See https://github.com/lightningnetwork/lnd/pull/5366#discussion_r686556441

Marking this comment as resolved and continue discussion there.